src/pages/app-development/app-builder-partnership-essentials.md

```diff
@@ -79,27 +118,35 @@ After partners get sandbox access, the Adobe Commerce engineering team will ask
 
 ## Distribution
 
-Before you distribute, you must be part of the [Technology Partner Program](#request-a-technology-partner-program-tpp-account).
+Before you distribute, you must be part of the [Technology Partner Program](#step-1-request-a-tpp-account).
```

Comment:
This line seems unnecessary. I don't think they'll even be able to build an app unless they're part of the program.